types/jquery/index.d.ts

```diff
@@ -4327,7 +4327,7 @@ interface JQuery<TElement = HTMLElement> extends Iterable<TElement> {
      * @see \`{@link https://api.jquery.com/insertAfter/ }\`
      * @since 1.0
      */
-    insertAfter(target: JQuery.Selector | JQuery.htmlString | JQuery.TypeOrArray<Element> | JQuery): this;
+    insertAfter<E extends Node>(target: JQuery.Selector | JQuery.htmlString | JQuery.TypeOrArray<E> | JQuery<E>): this;
```

Comment:
The type parameter is not necessary. You're effectively using it only once. This should be

```typescript
insertAfter(target: JQuery.Selector | JQuery.htmlString | JQuery.TypeOrArray<Node> | JQuery<Node>): this;
```